.github/workflows/periodic.yml

```diff
@@ -209,3 +209,30 @@ jobs:
         { include: [
           { config: "default", shard: 1, num_shards: 1, runner: "ubuntu-latest" },
         ]}
+
+  win-vs2019-cuda11_7-py3-build:
```

Comment:
We already have 11.8 GPU testing periodically that we plan to clean up.  Let's keep 11.8 instead and this can be removed.  I know that we are also moving to CUDA 12.1 at the moment